docs/astrocut/index.rst

```diff
@@ -256,7 +256,60 @@ The cutout target pixel file format is decribed `here <file_formats.html#target-
                 1  PIXELS        1 BinTableHDU    222   144R x 12C   [D, E, J, 25J, 25E, 25E, 25E, 25E, J, E, E, 38A]   
                 2  APERTURE      1 ImageHDU        45   (5, 5)   float64  
 
-  
+You can also create cutout target pixel files out of TESS cube files stored on MAST's AWS open data bucket.
+Using cube files stored on the cloud allows you the option to implement multiprocessing to improve cutout generation
+speed. To use multiprocessing, set the ``threads`` argument in ``cube_cut`` to an integer greater than 1. Alternatively, you
+can set set ``threads`` to ``"auto"``, which will set the number of threads based on the CPU count of your machine.
+Note that ``Total Time`` results may vary from machine to machine.
+
+.. code-block:: python
+
+                >>> from astrocut import CutoutFactory
+                >>> from astropy.coordinates import SkyCoord
+
+                >>> my_cutter = CutoutFactory()
+                >>> coord = SkyCoord(217.42893801, -62.67949189, unit="deg", frame="icrs")
+                >>> cutout_size = 30
+                >>> cube_file = "s3://stpubdata/tess/public/mast/tess-s0038-2-2-cube.fits"
+
+                >>> cut_factory.cube_cut(cube_file, coordinates=coord, cutout_size=cutout_size, verbose=True, threads="auto") #doctest: +SKIP
```

Comment:
Can we put some line breaks here so the `threads` option doesn't scroll off the side and is more easily visible?